### ISSUE TITLE
Use Maven for project

### ISSUE THREAD

**Issue by JarvisCraft** [SirCodalot/Vehicraft#1]
It would be a good sollution to use Maven for dependency build management as it would let other developers participate in the project with less pain.

**Comment by SirCodalot** [comment]
Yeah, I started using maven a long time ago, this project is just outdated.
